indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetDatastreamFilter.java

```diff
@@ -353,33 +367,6 @@ private void addDerivatives(List<Datastream> dsList, PID pid, boolean ownedByOth
         });
     }
 
-    /**
-     * Used to selectively add only thumbnail datastreams
-     *
-     * @param workObject the work object with the thumbnail relation
-     * @param datastreams work object's datastreams to add thumbnail streams to
-     */
-    private void addThumbnailDerivatives(WorkObject workObject, List<Datastream> datastreams) {
-        FileObject thumbnailObject = workObject.getThumbnailObject();
-
-        if (thumbnailObject != null) {
-            var updatedDatastreams = clearPreviousThumbnailDatastreams(datastreams);
-            addDerivatives(updatedDatastreams, thumbnailObject.getPid(), true, THUMBNAIL_DS_TYPES);
-        }
-    }
-
-    /**
-     *  There may be thumbnail streams from the primary object, so we'll clear those
-     *  before adding the assigned thumbnail datastreams
-     *
-     * @param datastreams full list of datastreams to index for the work object
-     * @return modified list of datastreams without thumbnail datastreams
-     */
-    private List<Datastream> clearPreviousThumbnailDatastreams(List<Datastream> datastreams) {
```

Comment:
I am wondering if we want a different version of this still for clearing previous jp2s, but we'll probably want to try it on the server first

Reply:
This doesn't seem to be an issue so I'm going to resolve this one